tests/backends/test_kafka.py

```diff
@@ -133,6 +144,70 @@ def test_auto_offset_reset_error(self) -> None:
                     consumer.poll(10.0)  # XXX: getting the subcription is slow
 
 
+def test_cooperative_rebalancing() -> None:
+    configuration = build_kafka_configuration(
+        {"bootstrap.servers": os.environ.get("DEFAULT_BROKERS", "localhost:9092")}
+    )
+
+    partitions_count = 2
+
+    group_id = uuid.uuid1().hex
+    producer = KafkaProducer(configuration)
+
+    consumer_a = KafkaConsumer(
+        {
+            **configuration,
+            "auto.offset.reset": "earliest",
+            "enable.auto.commit": False,
+            "enable.auto.offset.store": False,
+            "group.id": group_id,
+            "session.timeout.ms": 10000,
+        },
+        incremental_cooperative=True,
+    )
+    consumer_b = KafkaConsumer(
+        {
+            **configuration,
+            "auto.offset.reset": "earliest",
+            "enable.auto.commit": False,
+            "enable.auto.offset.store": False,
+            "group.id": group_id,
+            "session.timeout.ms": 10000,
+        },
+        incremental_cooperative=True,
+    )
```

Comment:
Thanks.
By the way, what happens if a number of consumers are incremental and others are not ? Which means what will happen at the first deployment ?

Reply:
Ok, let's do it one day I am on PTO. 
Out of curiosity, what happens if you start a consumer that works with the incremental rebalancing system to a group that is using the standard system ?